inferior-idris.el

```diff
@@ -368,4 +368,33 @@ call `ERROR' if there was an Idris error."
       ((:error condition &optional _spans)
        (message "Setting option %s to %s returned an error: %s." opt b condition)))))
 
+(defun idris-get-idris-version ()
+  "Ask the Idris compiler for its version information.
+Returns a cons cell whose car is a list of version number
+components and whose cdr is a list of prerelease identifiers, if
+applicable. Returns nil if the version of Idris used doesn't
+support asking for versions."
+  ;; The :version command was introduced at protocol version 2
+  (if (>= idris-protocol-version 2)
```

Comment:
Oh, I just bumped the version. It could also do a "on error nil" thing though. I'll go change it back.